.binder/postBuild

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# binder post build script
+set -ex
+
+(cd docs && make html)
+
+# uninstall docs requirements for a lighter docker image
+pip uninstall -y -r docs/requirements.txt
+
+# move examples to the notebooks folder
```

Comment:
the notebook execution (particularly the `execute_experiment` function) is very slow in binder, any idea why?

Reply:
I think it was as slow as a local run, what are the differences for you ?

Reply:
in local it takes less than a minute to execute, in the notebook the `execute_experiment` took more than 1 min for 1 cp task in the progress bar (I did not wait until the end, I can measure)

Reply:
7 min for the `execute_experiment`

Reply:
So I tested on my side, cp took 2:31 in notebook and 1:10 locally, I guess it's just because of a lower CPU and no SSD.
But I agree that even 1min is a bit too much, we could try to optimize the example (smaller model ?)

Reply:
The examples are already quite small, going from 1min to 7min is not normal
what are the resources allocated to the binder env?